.github/workflows/e2e.yaml

```diff
@@ -107,6 +107,7 @@ jobs:
         uses: dawidd6/action-download-artifact@v2
         with:
           github_token: ${{ secrets.SDK_TYPESCRIPT_ACTION_READ_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
Unrelated: Do we still need the github_token since the `sdk-typescript` repo is now public?

Reply:
probably not!